pywt/_extensions/_pywt.pyx

```diff
@@ -397,6 +399,67 @@ cdef public class Wavelet [type WaveletType, object WaveletObject]:
         def __set__(self, int value):
             self.w.biorthogonal = (value != 0)
 
+    property dwt_possible:
+        "DWT possible"
+        def __get__(self):
+            return bool(self.w.dwt_possible)
+        def __set__(self, int value):
+            self.w.dwt_possible = (value != 0)
+
+    property cwt_possible:
+        "CWT possible"
+        def __get__(self):
+            return bool(self.w.cwt_possible)
+        def __set__(self, int value):
+            self.w.cwt_possible = (value != 0)
```

Comment:
I think this class should be refactored to separate out CWT and DWT wavelets. Something like this:

```
class Wavelet:
  # common properties
  ...

class DiscreteWavelet(Wavelet):
  # DWT specific things
  ...

class ContinuousWavelet(Wavelet):
  # CWT specific things
  ...
```

If a method is available for both classes, but can't be defined without knowing whether the class is continuous or discrete, define it on `Wavelet`, but use `abc.abstractmethod` if it's available in Cython, or raise `NotImplementedError` otherwise.

Hopefully this will show what can be split over two separate C structs as well.

I realize this could be a bit complicated so I'm happy to answer any questions. I'm able to respond to things more quickly again now so it shouldn't take too long.